config/sara/sara.yml

```diff
@@ -53,3 +53,7 @@ max_user_filters: <%= ENV["MAX_USER_FILTERS"] || 150 %>
 # How long to wait for downloads to complete // how long to wait before downloads are cleaned up after clicking the download link
 # in minutes
 download_destroy_wait_time: <%= ENV['SARA_ALERT_DOWNLOAD_DESTROY_WAIT_TIME']|| 60 %>
+
+# The date when case development starts being tracked (for analytics and exports)
+# This date should be set to the date that the AddReportedAtToAssessments migration is applied, which should be when application version 1.38 is deployed
+case_development_analytics_start_date: 11/2/2021
```

Comment:
This comment could use a little more detail, and the value should be fetched from an environment variable:
```suggestion
# The date when case development starts being tracked (for analytics and exports).
# This date should be set to the date that the AddReportedAtToAssessments migration is applied, which should be when application version 1.38 is deployed.
# This date is used on the Analytics page to indicate to users when cumulative numbers start from.
# If your Sara Alert instance was initially deployed at version 1.38 or later, set this to empty string.
case_development_analytics_start_date: <%= ENV['CASE_DEVELOPMENT_ANALYTICS_START_DATE'] || '' %>
```
and then also add to `.env-prod-enrollment-example`:
```ini
CASE_DEVELOPMENT_ANALYTICS_START_DATE=11/2/2021
```
We will the separately tell APHL to set that environment variable.

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1085/commits/3a713d3d7ba105e098cbf811b449fb506dbd5f49